iocore/net/SSLNetVConnection.cc

```diff
@@ -1260,9 +1260,9 @@ SSLNetVConnection::sslServerHandShakeEvent(int &err)
       this->reenable(&read.vio);
       this->read.triggered = 1;
     }
```

Comment:
Go through to the switch statement at below and return SSL_HANDSHAKE_WANT_READ,  then the `net_read_io()` will reenable vc and set read state properly.